doc/examples/audiopool/lib/main.dart

```diff
@@ -11,17 +11,16 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 
 void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
   final Size size = await Flame.util.initialDimensions();
   final MyGame game = MyGame(size);
   runApp(game.widget);
-
-  final TapGestureRecognizer taps = TapGestureRecognizer()
-    ..onTapDown = (_) => game.tap();
-  Flame.util.addGestureRecognizer(taps);
 }
 
-TextConfig regular = TextConfig(color: BasicPalette.white.color);
-AudioPool pool = AudioPool('laser.mp3');
+final regular = TextConfig(color: BasicPalette.white.color);
+final pool = AudioPool('laser.mp3');
+final black = BasicPalette.black.paint;
```

Comment:
Keep PRs like this as dry as possible.

Reply:
I can split, I just cannot stand linting issues while I was looking at each example code to see if it was working.